modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/search/spi/model/index/contributor/CPConfigurationEntryModelDocumentContributor.java

```diff
@@ -42,18 +49,62 @@ public void contribute(
 			document.addKeyword(
 				CPField.CP_CONFIGURATION_LIST_ID,
 				cpConfigurationEntry.getCPConfigurationListId());
-			document.addNumber(CPField.DEPTH, cpConfigurationEntry.getDepth());
+
 			document.addNumber(
-				CPField.HEIGHT, cpConfigurationEntry.getHeight());
+				CPField.CP_CONFIGURATION_LIST_IDS,
+				TransformUtil.transformToArray(
+					_cpConfigurationListLocalService.getCPConfigurationLists(
+						cpConfigurationEntry.getGroupId(),
+						cpConfigurationEntry.getCompanyId()),
+					cpConfigurationList -> {
+						if (cpConfigurationList.getCPConfigurationListId() ==
+								cpConfigurationEntry.
+									getCPConfigurationListId()) {
+
+							return null;
+						}
+
+						return cpConfigurationEntry.getCPConfigurationListId();
```

Comment:
yes I need to fw this pr otherwise it will get to much bigger, the idea is to add the new column that will hold all the ids and do the reindex with that value